### ISSUE TITLE
Check time for AFK

### ISSUE THREAD

**Issue by josellyto** [AthenaEventEngine/AthenaCore#195]
When a character is kikado of the event due to inactivity generates errors in GameServer. and if all of them are, the buga event does not end.

**Comment by Lucheti29** [comment]
Yes, the engine is not handling the "force leave" correctly. It'll take a bit time to fix this.
